HW02_A_Graph-Fails.Rmd

```diff
@@ -1,87 +1,100 @@
+
+It is also best coding practice to only call the packages you use, so if you use a package but end up tossing the code you use for it, then make sure to remove loading it in the first place. For example, I could use `library("tidyverse")` but since this script will only be using ggplot2, I only load ggplot2.
+
+```{r load libraries}
+library("ggplot2")
+library("magrittr") #so I can do some piping
+```
+
+
+### Graph Fail 1
+
+What error is being thrown? How do you correct it? (hint, the error message tells you)
+
+Nikita - The first error is that we used %>% instead of +? 
+
+```{r}
+data(mpg) #this is a dataset from the ggplot2 package
+
+mpg %>% 
+  ggplot(mapping = aes(x = city, y = hwy, color = "blue")) %>% 
+  geom_point()
+
+```
+
+### Graph Fail 2
+Why aren't the points blue? It is making me blue that the points in the graph aren't blue :`(
+
+Nikita - I fixed the code! Essentially, the points weren't showing up as blue because the color argument was inside the 'aes'. It should have been outside the aes! 
+
+```{r}
+ggplot(data = mpg) + 
+  geom_point(color= 'blue', mapping = aes(x = displ, y = hwy)) 
+
+```
+
+
+### Graph Fail 3
+Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
+
+Second, I wanted to move the legend on top of the graph since there aren't any points there, putting it at approximately the point/ordered pair (5, 40). How do you actually do this? Also, how do you remove the legend title ("class")? Finally, how would you remove the plot legend completely?
+
+Nikita - Alpha changes the transparency of the points, and setting it to 2 makes it so that the graph won't show up at all since opacity is on a 0-1 scale. If you want the points to be bigger, you should add a size argument. Using the function, theme, will allow you to change the position. Instead of 5, 40, you should change it to "top". That will move the legened to the top. If you want to remove the legend title, say theme(legend.title)=element_blank()). If you want to remove the legend entirely, say theme(legend.position='none'). 
+
+```{r}
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), alpha = 1) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = c(5, 40))
+
+```
+
+### Graph Fail 4
+I wanted just one smoothing line. Just one line, to show the general relationship here. But that's not happening. Instead I'm getting 3 lines, why and fix it please?
+
+Nikita -I have been trying to figure this out, but I am not sure. 
```

Comment:
I added this code "geom_smooth(se = F)"  the "se = F" makes it so it won't show the error in the line of fit I think